### ISSUE TITLE
getSigningCertificate() promise is not resolving

### ISSUE THREAD

**Issue by Sanglepp** [web-eid/web-eid.js#21]
I am trying to use getSigningCertificate() method, defined [here](https://github.com/web-eid/web-eid.js/blob/f940dcea130402d65d19ba54dfb42f7d5411f88d/src/web-eid.ts#L132).

Currently the action opens up the user interaction modal to choose the certificate but the promise never resolves. 

`
   const webeId = require("node_modules/@web-eid/web-eid-library/web-eid");

    const { certificate } = await webeId.getSigningCertificate({ lang: 'et' });
    console.log(certificate);
`

My goal is to get the certificate to read the public data of the id-card for customer search with id-card.

Am I missing something or the getSigningCertificate() function resolve with `{
    certificate,
    supportedSignatureAlgorithms,
  };`
  
  I am using the 2.0.0 version of this library

**Comment by taneltm** [comment]
Which browser and OS are you using?  

What do you get with the following code:
```js
try {
  console.log(await webeId.status());
} catch (e) {
  const { name, message, library, extension, nativeApp, requiresUpdate } = e;
  console.log({ name, message, library, extension, nativeApp, requiresUpdate });
}
```

**Comment by Sanglepp** [comment]
Hi and thanks for the quick reply!

I didn't use the status request yesterday. Today the certificate request didn't initiate the certificate UI in native app and I got an error from the status request that I have mismatched versions.

Seems that the native web-eid app has been updated quite recently. After the update I get the correct response from getSigningCertificate() method.

Thanks again for the answer and for the recent updates. I will close the issue and will implement the status check to my app logic to prevent future confusions and issues.
